files/miscfiles/newsfile

```diff
@@ -17,6 +17,6 @@ the talker (instead of readin' this generic thing!)
 To find out what all the changes are it's best to talk to your admins and
 ask them all that has changed by telling them to look in the amnuts_changes
 file that came with Amnuts.  Of course, you could also check out the
-Amnuts website at: ~OL~FChttp://amnuts.talker.com/~RS if you want!
+Amnuts website at: ~OL~FChttps://www.amnuts.net/~RS if you want!
```

Comment:
It is good to know amnuts.net still works (although it seems to just be a redirect) and even better seems to now support https encryption but I notice you did not change everything to point the the https. I also noticed the redirector does not seem to care if the URL has the leading "www." or not.
There are many http://www.amnuts.net/ about vs. https://www.amnuts.net/
It seems to me, if you intend to keep the amnuts.net redirector around, all the references should be changed to the same thing and despite most websites today redirecting example.com to www.example.com, since it is only a redirector I would prefer the simplicity of: https://amnuts.net/

Reply:
Fixed in https://github.com/amnuts/Amnuts/pull/12/commits/a7ce2cf4d9560297cff7e2c297f1e50582793746

Reply:
Found some of the old site designs, btw 😁  https://github.com/amnuts/Amnuts/discussions/13